bric.c

```diff
@@ -1375,6 +1388,99 @@ void init_editor(void)
         Editor.screen_rows -= 2; // get room for status bar
 }
 
+void load_config_file(void)
+{
+    struct passwd *user = getpwuid(getuid());
+    char config_file[80];
+    config_file[0] = 0;
+    strcat(config_file, user->pw_dir);
+    strcat(config_file, "/.bricrc");
```

Comment:
No problem that's cool, I see how using pwd works now. Nice job:) And thanks for the example config file